### ISSUE TITLE
Cameras detected, but are blank.

### ISSUE THREAD

**Issue by lawesome1444** [cboulay/PSMoveService#426]
My PS3 Eye's seem to not display anything. I've tried moving the USB cable around the sockets from Motherboard to the front but nothing. I've also uninstalled and re-installed the drivers for the Eyes multiple times now, but still they show blank. The only time I can get some working is if I plug all 4 of them in, but its still random if it will work.

**Comment by cboulay** [comment]
Did you try test_camera.exe?

**Comment by lawesome1444** [comment]
@cboulay Yes, I have tried both test_camera.exe and test_camera_parrellel.exe, still nothing
Apologies for late reply, I was asleep

**Comment by lawesome1444** [comment]
@cboulay In a little more detail, both camera popups appear, but show nothing, just gray. Further more, one of the camera windows is small, the other bigger,
Sometimes I can get one working, but it will stop working after that.
(btw this is all problematic in the test_camera and text_camera_parallel software)

**Comment by cboulay** [comment]
Do you have access to a Linux or Mac computer? The cameras work much better there (no driver installation necessary) so they can serve as a good test to see if it's a problem with the camera hardware or just driver installation.

**Comment by lawesome1444** [comment]
@cboulay Sadly, not. Neither a PS3 to also check. Anything else that could help? Thanks!

**Comment by JxnBld** [comment]
I can't even FIND test_camera.exe or any .exe files downloaded from the instructions in the readme wiki page.

**Comment by lmnope1** [comment]
Hello I have a similar issue. Test camera .exe shows both of my cameras well, but in the PSmoveconfigtool they display as a white screen
